frontend/institution/institutionService.js

```diff
@@ -120,13 +120,15 @@
             return deffered.promise;
         };
 
-        service.removeInstitution = function removeInstitution(institutionKey, removeHierarchy) {
+        service.removeInstitution = function removeInstitution(institutionKey, removeHierarchy, justification) {
             var deffered = $q.defer();
-            $http.delete(INSTITUTIONS_URI + "/" + institutionKey + "?removeHierarchy=" + removeHierarchy).then(function success(info) {
-                deffered.resolve(info.data);
-            }, function error(data) {
-                deffered.reject(data);
-            });
+            $http.delete(INSTITUTIONS_URI + "/" + institutionKey + "?removeHierarchy=" + removeHierarchy + "&justification=" + justification).then(
```

Comment:
Usa o padrão do angular para Query Params, que usa json para fazer esse envio dos parâmetros ao invés de usar concatenação de strings.

https://stackoverflow.com/questions/13760070/angularjs-passing-data-to-http-get-request